Sources/SpeziLLMLocal/Mock/LLMLocalMockSession.swift

```diff
@@ -1,111 +0,0 @@
-//
-// This source file is part of the Stanford Spezi open source project
-//
-// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
-//
-// SPDX-License-Identifier: MIT
-//
-
-import Foundation
-import Observation
-import SpeziLLM
-
-
-/// A mock ``LLMLocalMockSession``, used for testing purposes.
-///
-/// See `LLMMockSession` for more details
-@Observable
-public final class LLMLocalMockSession: LLMSession, @unchecked Sendable {
```

Comment:
Technically that would be considered a breaking change and we would need to tag a 0.10.0 for that.

Reply:
I agree, we could wait until #64 is merged and then tag a 0.10.0